xml/cl.xml

```diff
@@ -1633,7 +1631,6 @@ server's OpenCL/api-docs repository.
         <enum value="0x1108"        name="CL_MEM_OFFSET"/>
         <enum value="0x1109"        name="CL_MEM_USES_SVM_POINTER"/>
         <enum value="0x110A"        name="CL_MEM_PROPERTIES"/>
-        <enum value="0x110B"        name="CL_MEM_DEVICE_ADDRESS_EXT"/>
```

Comment:
I would be fine with both options: keeping this enum here, and moving it down to your new block.

But note: you've changed the `<unused>` block on the next line when adding enums, but you didn't revert it when removing your enums from here.

Reply:
Thanks. Fixed.